src/model/race.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
```

Comment:
Do you typically check in the generated JS files?

Reply:
Well I deleted it :P  Seemed wrong to check it in

Reply:
We can, or not. Either way. I think it is cleaner to leave it out.